google_checks_1.8.xml

```diff
@@ -58,9 +58,7 @@
                       value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
         </module>
         <module name="NeedBraces"/>
-        <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
```

Comment:
The line length check is still configured above. This was a property that apparently was deprecated for a long time and was finally removed in checkstyle 8.